### PR TITLE
feat: add RUST_LOG support

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

File: operator/src/network/ceramic.rs [hunk 1]
```diff
@@ -164,6 +164,8 @@ pub struct RustIpfsSpec {
     pub image_pull_policy: Option<String>,
     // Resource limits for ipfs nodes, applies to both requests and limits.
     pub resource_limits: Option<ResourceLimitsSpec>,
+    /// Value of the RUST_LOG env var.
+    pub rust_log: Option<String>,
```
Comment (3benbox) [inline operator/src/network/ceramic.rs:168]: What happens on an invalid string?
Does it use default, crash, not log?

Comment (3benbox) [reply operator/src/network/ceramic.rs:168]: invalid meaning somehting like `"invalid-logs-level,ceramic_onehundred=debug,tracing_actix_web=all"`

Comment (nathanielc) [reply operator/src/network/ceramic.rs:168]: Defaults to `info` for all logs.

**Review by 3benbox — APPROVED** [review]
lgtm

**Pull request merged**